OutOfSchool/OutOfSchool.WebApi/appsettings.Development.json

```diff
@@ -76,5 +76,15 @@
     "ClientId": "aikom",
     "ClientSecret": "replace_me",
     "TokenEndpoint": "http://localhost:8083/oauth2/token"
+  },
+  "SportsRegistryApiClient": {
+    "Enable": true,
+    "ApiUrl": "https://external-service-api-mms-main.apps.krrt-stage.ncr.gov.ua/",
+    "TokenEndpoint": "https://mms-keycloak.apps.krrt-stage.ncr.gov.ua/auth/realms/mms-external-system/protocol/openid-connect/token",
+    "ClientId": "pozashkillya",
+    "ClientSecret": "replace_me"
+  },
+  "InstitutionOptions": {
+    "MinistryOfSportId": "b67a4f29-728e-4bb0-bb42-4a9d7e0bd90a"
```

Comment:
as we use same value, no need to repeat everywhere, regular `appsettings.json` is fine